projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/active-integrations/index.js

```diff
@@ -28,6 +29,19 @@ export default function ActiveIntegrations( { integrations, attributes, isLoadin
 					} );
 				}
 				break;
+			case 'salesforce':
+				if (
+					attributes.salesforceData?.sendToSalesforce &&
+					attributes.salesforceData?.organizationId &&
+					/^[a-zA-Z0-9]{15,18}$/.test( attributes.salesforceData.organizationId.trim() )
+				) {
+					acc.push( {
+						...integration,
+						icon: <SalesforceIcon width={ 30 } height={ 30 } />,
+						tooltip: __( 'Salesforce is connected for this form', 'jetpack-forms' ),
+					} );
+				}
+				break;
```

Comment:
When Salesforce ID is added, and is turned on, we'll show the active salesforce ID in the Integrations sidebar panel.